.travis.yml

```diff
@@ -54,9 +54,9 @@ install:
 
 script:
   - if [[ $TEST_TARGET == 'default' ]]; then
-      cp -r tests /tmp && cd /tmp ;
-      pytest -vv tests ;
+      cp -r ogh/tests /tmp && cd /tmp ;
+      pytest -vv test_ogh.py ;
```

Comment:
You should keep the tests folder, rather than pytest a specific file

Reply:
@lsetiawan 
Should that be coded as follows, to run pytest on files that say 'test' within the tmp folder?

cp -r ogh/tests /tmp && cd/tmp ;
pytest -vv test*

Reply:
if you want the specific file to run just do `pytest -vv tests/test_ogh.py`.